packages/tools/viewer/src/viewer.ts

```diff
@@ -493,13 +533,35 @@ export class Viewer implements IDisposable {
     }
 
     /**
-     * A value between 0 and 1 that specifies how much to blur the skybox.
+     * Get the current environment configuration.
      */
-    public get skyboxBlur(): number {
-        return this._skyboxBlur;
+    public get environmentConfig(): Readonly<EnvironmentParams> {
+        let values = {
+            blur: this._skyboxBlur,
+        } as EnvironmentParams;
+        if (this._reflectionTexture) {
+            values = { ...values, intensity: this._reflectionsIntensity, rotation: this._reflectionsRotation };
+        }
+        return values;

```

Comment:
This property returns an EnvironmentParams object, which requires all properties. Shouldn't this just be:
```suggestion
    return {
        intensity: this._reflectionsIntensity,
        blur: this._skyboxBlur,
        rotation: this._reflectionsRotation,
        visible: this._skybox?.getEnabled() ?? false,
    };
```